pom.xml

```diff
@@ -54,11 +54,11 @@ under the License.
 		<maven.compiler.target>${java.version}</maven.compiler.target>
 		<slf4j.version>1.7.36</slf4j.version>
 		<log4j.version>1.2.17</log4j.version>
-		<jmh.version>1.19</jmh.version>
+		<jmh.version>1.37</jmh.version>
```

Comment:
Needed a newer version for async-profiler support in JMH so went for the most recent release.